lib/galaxy/tool_util/linters/_util.py

```diff
@@ -1,5 +1,56 @@
 import re
 
+from Cheetah.Compiler import Compiler
+from Cheetah.Template import Template
```

Comment:
https://github.com/galaxyproject/galaxy/blob/dev/packages/tool_util/setup.cfg#L34 this needs to be updated to require ``Cheetah3`` or ``galaxy-util[template]``.

Reply:
Thanks for the hint @jmchilton added `Cheetah3`